tools/docker/docker-compose-dev.yaml

```diff
@@ -252,7 +252,7 @@ services:
     restart: always
 
   eda-webhook-api:
-    image: "${EDA_IMAGE:-quay.io/ansible/eda-server:main}"
+    image: ${EDA_IMAGE:-localhost/aap-eda}
```

Comment:
Is this change picked by accident

Reply:
good catch

Reply:
the development image should use the local image by default. The rest of services do the same

Reply:
This was included with the rest of the changes because they are all the result of debugging towards the same goal. I believe using the default image was necessary to get updated DAB, and all the default settings that come with that. This was copied from other services in the same file, as @Alex-Izquierdo is saying.